samples/ipc_telemetry/producers/sensors_v1i1_interface.c

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from sensors v1 DL.
+ *
+ * Implement the code under the concrete functions.
+ *
+ ********************************************************************/
+
+#include "sensors_v1i1_interface.h"
+#include "az_ulib_capability_api.h"
+#include "az_ulib_descriptor_api.h"
+#include "az_ulib_ipc_api.h"
+#include "az_ulib_result.h"
+#include "sensors_1_model.h"
+#include "sensors_v1i1.h"
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static az_result sensors_1_subscribe_temperature_concrete(
+    const az_ulib_telemetry_subscribe_model* const in,
+    az_ulib_model_out* out)
+{
+  (void)out;
+  return sensors_v1i1_subscribe_temperature(in->context, in->callback);
+}
+
+static az_result sensors_1_unsubscribe_temperature_concrete(
+    const az_ulib_telemetry_subscribe_model* const in,
+    az_ulib_model_out* out)
+{
+  (void)out;
+  return sensors_v1i1_unsubscribe_temperature(in->context, in->callback);
+}
+
+static az_result sensors_1_subscribe_accelerometer_concrete(
+    const az_ulib_telemetry_subscribe_model* const in,
+    az_ulib_model_out* out)
+{
+  (void)out;
+  return sensors_v1i1_subscribe_accelerometer(in->context, in->callback);
+}
+
+static az_result sensors_1_unsubscribe_accelerometer_concrete(
+    const az_ulib_telemetry_subscribe_model* const in,
+    az_ulib_model_out* out)
+{
+  (void)out;
+  return sensors_v1i1_unsubscribe_accelerometer(in->context, in->callback);
+}
+
+static az_result sensors_1_get_temperature_interval_concrete(
+    const sensors_1_temperature_interval_model* const in,
+    sensors_1_temperature_interval_model* out)
+{
+  (void)in;
+  *out = sensors_v1i1_get_temperature_interval();
+  return AZ_OK;
+}
+
+static az_result sensors_1_set_temperature_interval_concrete(
+    const sensors_1_temperature_interval_model* const in,
+    sensors_1_temperature_interval_model* out)
+{
+  sensors_v1i1_set_temperature_interval(*in);
+  if (out != NULL)
+  {
+    *out = sensors_v1i1_get_temperature_interval();
+  }
+  return AZ_OK;
+}
+
+static az_result sensors_1_get_accelerometer_interval_concrete(
+    const sensors_1_accelerometer_interval_model* const in,
+    sensors_1_accelerometer_interval_model* out)
+{
+  (void)in;
+  *out = sensors_v1i1_get_accelerometer_interval();
+  return AZ_OK;
+}
+
+static az_result sensors_1_set_accelerometer_interval_concrete(
+    const sensors_1_accelerometer_interval_model* const in,
+    sensors_1_accelerometer_interval_model* out)
+{
+  sensors_v1i1_set_accelerometer_interval(*in);
+  if (out != NULL)
+  {
+    *out = sensors_v1i1_get_accelerometer_interval();
+  }
+  return AZ_OK;
+}
+
+static const az_ulib_capability_descriptor SENSORS_1_CAPABILITIES[]
```

Comment:
Something similar here:
```
/* ***** Auto-generated code. Do not change. ***** */
static const az_ulib_capability_descriptor SENSORS_1_CAPABILITIES[]
{
...
}
/* ***** End auto-generated code. ***** */
```

Reply:
I don't think we need it once this entire file shall be autogenerated.